manifest.json

```diff
@@ -1,20 +1,28 @@
 {
-  "description": "A basic template for a Mozilla Rally Study.",
-  "author": "Mozilla",
+  "description": "Local News Demand Study",
+  "author": "Stanford GSB",
   "manifest_version": 2,
-  "name": "Rally Basic Study Template",
-  "version": "0.3.1",
-  "homepage_url": "https://github.com/mozilla-ion/ion-basic-study",
-
-  "applications": {
-    "gecko": {
-      "id": "ion-basic-study@mozilla.org",
-      "strict_min_version": "77.0a1"
-    }
-  },
-
-  "permissions": [],
+  "name": "Local News Demand Study",
+  "version": "0.1",
+  "homepage_url": "https://github.com/bchivers-stanford/local_news_demand",
 
+  "permissions": [
+    "<all_urls>",
+    "idle",
+    "notifications",
+    "storage",
+    "tabs",
+    "unlimitedStorage",
+    "webRequest",
+    "alarms",
+    "telemetry",
+    "history",
+    "webNavigation",
+    "webRequest",
+    "webRequestBlocking"
+  ],
+  "browser_action": {},
+  "content_security_policy": "script-src 'self' 'sha256-csyiOLMfXk2f5pU99mqYFyshgnVYbdp6o9bnQ9hntPA=' 'sha256-nYNRfLKTaKqgi4+CK/mcp9hdSsmD8F17GWuo+vQGfqU='; object-src 'self'",
   "background": {
     "scripts": ["dist/background.js"]
   }
```

Comment:
There should be an icon too, so it's not the default "puzzle piece".

Reply:
Added an icon